druid/src/env.rs

```diff
@@ -208,6 +208,25 @@ impl Env {
             .map(|value| value.to_inner_unchecked())
     }
 
+    /// Gets a value from the environment, in it's encapsulated [`Value`] form,
```

Comment:
~it's~ its

Reply:
Silly inconsistent grammar in my native language 😬
Should be fixed.